File: TypeScript/pangram.js
var readline = require('readline');
var rl = readline.createInterface({ input: process.stdin, output: process.stdout });
rl.question('', function (sentence) {
    console.log(check(sentence.trim()));
    rl.close();
});
function check(sentence) {
    var letters = new Map();
    for (var _i = 0, sentence_1 = sentence; _i < sentence_1.length; _i++) {
        var ch = sentence_1[_i];
        letters.set(ch, (letters.get(ch) || 0) + 1);
    }
    var isPan = true;
    for (var code = 97; code <= 122; code++) {
        var c = String.fromCharCode(code);
        if (!letters.has(c) || letters.get(c) === 0) {
            console.log("There is no ".concat(c));
            isPan = false;
        }
    }
    return isPan;
}

File: Golang/pangram.go
package main

import(
	"fmt"
	"os"
	"bufio"
)

func main(){
	var sentence string = input()
	output(sentence)
}

func input() string{
	scanner := bufio.NewScanner(os.Stdin)
	scanner.Scan()
	return scanner.Text()
}

func output(sentence string){
	var result bool = check(sentence)
	fmt.Printf("%t", result)
}

func check(sentence string) bool{
	var letters map[byte]int = make(map[byte]int)
	for i := 0; i < len(sentence); i++{
		letters[sentence[i]]++
	}
	var c byte = 97
	var isPan bool = true
	for ; c <= 122; c++{
		if letters[c] == 0{
			fmt.Printf("There is no %c\n", c)
			isPan = false
		}
	}
	return isPan
}